x-pack/plugins/enterprise_search/server/lib/analytics/delete_analytics_collection.ts

```diff
@@ -10,20 +10,9 @@ import { IScopedClusterClient } from '@kbn/core-elasticsearch-server';
 import { ANALYTICS_COLLECTIONS_INDEX } from '../..';
 
 import { ErrorCode } from '../../../common/types/error_codes';
-import { fetchIndices } from '../indices/fetch_indices';
 
 import { fetchAnalyticsCollectionByName } from './fetch_analytics_collection';
 
-const deleteAnalyticsCollectionEvents = async (client: IScopedClusterClient, name: string) => {
-  const indexPattern = `elastic_analytics-events-${name}-*`;
-  const indices = await fetchIndices(client, indexPattern, true, false);
-
-  await client.asCurrentUser.indices.delete({
```

Comment:
Should we put a ticket in the backlog to delete the documents instead? Since we cant delete the indices setup by the datastream.